examples/getting_started/2_Pipeline.ipynb

```diff
@@ -500,7 +651,31 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Usually, however, such custom operations are done directly on the aggregate during the ***Transformation*** stage; the `how` operations are meant for simple, well-defined transformations solely for the final steps of visualization, which allows the main aggregate array to stay in the original units and scale in which it was measured.  Using `how` also helps simplify the subsequent ***Embedding*** stage, letting it provide one of a fixed set of legend types, either linear (for `how=linear`), logarithmic (for `how=log`) or percentile (for `how=eq_hist`).  See the [shade docs](https://datashader.org/api.html#datashader.transfer_functions.shade) for more details on the `how` functions. \n",
+    "Because Datashader is designed to reveal the properties of your dataset without making any assumptions about its range or form, the default transformation is `eq_hist`; you can select any other `how` option for `tf.shade` if needed and appropriate for your particular dataset.\n",
```

Comment:
```suggestion
    "Because Datashader is designed to reveal the properties of your dataset without making any assumptions about its range or form, the default transformation is `eq_hist`; you can select any other `how` option for `tf.shade` if you know it is appropriate for your particular dataset.\n",
```